Developing-an-Interactive-Educational-Platform-with-Directus.md

```diff
@@ -0,0 +1,374 @@
+
+Going further Time limit field, which will be a text, and passing score which will be text but an integer, setting the minimum value to 0 and maximum to one hundred, or ten depending on your grade, in this blog we will leave it 100.
+
+![](https://lh7-us.googleusercontent.com/3RWuxpL3dlE8OO3y9iKCt1yJOGOTt9_XA3rILELxhjMhI3IZn50A1wCXZXBcNnEiL6pPnuLANlFz61Yn3gzzR0VGuHiZFGNVcoOTJ7BdaNIbTDpqdno2hUAF65gFTcb0YIy228LRj5lEnGQli1MwrUk)
+
+Lastly, we will create the question collection and for the field, we will be using the WYSIWYG option.
+
+### Instructor Collection,
+
+Every Course is supposed to have someone with experience to teach it. We will go ahead to create its collection, we will name the collection, and select the optional field as usual. In our collection we will have the; Instructor's Name, Email, Bio, Profile Picture or Avatar, and Courses Taught which will be linked to the Course collection.
+
+For our name and email, we will be using the Text field, for the Bio we will be using the WYSIWYG field, and for the avatar, we will be using the image field.
+
+All of these fields have previously been demonstrated, we can go ahead to create them. 
+
+### ![](https://lh7-us.googleusercontent.com/OQ8roD49L-w080wFaGX1OAlNPJz4pHKeE3L4VZt9ZdcxE1jXoO3JQI8kbc9TQv91WyZtax0cUvp_dyZDFrQ102PkIfPlQqp-ZrC8T04usmMkJ_AKCeXlo2JF_ZEpvb4Y4Bb-3XJMU7mz5iaaLHYVebo)
+
+### Enrollment Collection
+
+The enrollment would be kind of tricky as it deals with linking a few of the other collections, the fields will look like this:
+
+-   Enrollment Date: Date Field 
+
+-   Status: Text or Select Field 
+
+-   Grades/Progress: Text or Numeric Field
+
+We will go ahead to create the enrollment date, status, and grades placing them in their proper fields.
+
+This is what the Collections look like:
+
+![](https://lh7-us.googleusercontent.com/LwtV7Fv9iNLaFrbI2QbXvnXayspC7A7np3txo0PsMSpyG08RzEWPU6msdMuZgxOAHN87HQuI6DH8K8ZpD0G8h20K_T_la5cRoLcvEQe9Xsw3z-VZ-eFHYi57cnLFuAJykvuvUEFuKEXuhZm_Zt-uKwk)
+
+We could give it nice icons to make them look better and interactive. To do this on any collection, click any of your choices and scroll below the fields created, you will find the Icon option, and pick anyone that fits, Directus provides us with a lot of options, we can put in random names and get random icons too as seen below.
+
+![](https://lh7-us.googleusercontent.com/Amg-dwA9bXk2IH0GYgr3ncSC_HnUIQJZRa7GTm-_h5O75w8dNdoJacK1YGDCUYyAxC2J17nThYDWC7PD54Pp1BhMP81KuMyCgICRC9Xws8sBbkugmNyIH30Fl9dWetvz54pTaCKOiFmNvcDwhRd7wb8)
+
+We can go ahead, and add the icons it should look like the image below.
+
+![](https://lh7-us.googleusercontent.com/j-0FOXFC2FN0v1U7SdNwIoEv1d3QIzVtnxTpm9a3_WQUbRVN2HuaKyzRMNuBfZ3RkNAVe_pV7LmdqshdakNguHFdCIJosJHi6r1i_p0AvD9gRLWUQtrtP8d33LWU-OPX9gHmne2jBMzPjhQrG608xT0)
```

Comment:
Remove

Reply:
@phazonoverload What exactly am I taking off, the whole section?

Reply:
Yup. Lines 69 to 79 where the comment was :) It's not important to what you're trying to teach - it's either a visual validation of the steps above, or UI customization